graph.py

```diff
@@ -0,0 +1,75 @@
+from matplotlib import pyplot as pl
+from dateutil import parser
+import calendar
+import numpy as np
+import csv
+import sys
+
+'''
+The 'fname' sent to this module should be the name of a file which
+has comma separated values in the format: MM-DD-YYYY, value
+E.g:
+
+12-10-2014, 5
+12-11-2014, 11
+12-12-2014, 14
+12-13-2014, 19
+12-14-2014, 23
+
+The 'value' is nothing but the GiB used until that date.
+
+Access this module as given below:
+
+plotgraph(<filename>)
+
+'''
+
+#This function accepts file name and return two vectors containing date and value separately.
```

Comment:
Use Docstring instead of Comments to describe about a methods. You can follow Spinx format